autoblocks/_impl/testing/run.py

```diff
@@ -141,7 +145,7 @@ async def run_test_case_unsafe(
             testCaseOutput=serialize(output),
         ),
     )
-
+    current_test_case_hash.reset(token)
```

Comment:
this isn't necessary, but more of a safety mechanism

Reply:
it's also possible this wouldn't be reached if there is an uncaught error in `fn`, you'd have to put this in a `finally` block

Reply:
hmm ya good call, let me move it up a level

Reply:
i don't think it matters too much since it's unnecessary like you said, was just a thought

Reply:
ya it doesn't matter for this use case, but good hygeine in case we use more and people reference this, I did the same for the test id too.

Reply:
ok nice